models/issues/pull.go

```diff
@@ -315,6 +316,25 @@ func (pr *PullRequest) LoadHeadRepo(ctx context.Context) (err error) {
 	return nil
 }
 
+func (pr *PullRequest) LoadRequestedReviewers(ctx context.Context) (err error) {
+	// Reset maybe preexisting reviewers
+	pr.RequestedReviewers = []*user_model.User{}
```

Comment:
Why resetting it? Maybe just return if it's length is great than 0?

Reply:
I referred to `models/issues/assignees.LoadAssignees`, and it seems that it should be named "Reload" instead. Currently, it is only used for compatibility with unit tests.

Reply:
Done.